app/lib/evss/claims_service.rb

```diff
@@ -0,0 +1,33 @@
+module EVSS
```

Comment:
Preface this by saying I'd like to keep the namespaces as short as possible, but do we want all services to live in app/lib/services? There will be an mvi service soon and I'm sure many more.